KopernicusExamples/Creating New Bodies/Star/Nemesis.cfg

```diff
@@ -2,7 +2,7 @@
 // Outdated
 // Contains Monsters
 
-@Kopernicus 
+@Kopernicus:NEEDS[!KOPERNICUS]
```

Comment:
The config *should* still work, except for the SolarPowerCurve. Thats useless but not parsed anyway. If it still works we should keep it and just remove the SolarPowerCurve.